.circleci/config.yml

```diff
@@ -35,6 +35,23 @@ workflows:
   pipeline:
     jobs:
     - tests:
+        name: buster-12
         dist: buster
         pgversion: "12"
+    - tests:
+        name: stretch-96
+        dist: stretch
+        pgversion: "9.6"
+    - tests:
+        name: centos7-12
+        dist: centos7
+        pgversion: "12"
+    - tests:
+        name: centos7-10
+        dist: centos7
+        pgversion: "10"
+    - tests:
+        name: centos6-96
+        dist: centos6
+        pgversion: "9.6"
```

Comment:
Pas de 9.5 ?

Reply:
On avait noté les versions à implémenter dans la feuille de route.
J'ai repris sans me poser de questions.
Je change pour tester aussi la 9.5.